src/firefox/remote.js

```diff
@@ -143,38 +143,35 @@ export class RemoteFirefox {
       });
   }
 
-  checkForAddonReloading(
+  async checkForAddonReloading(
     addon: FirefoxRDPAddonActor
   ): Promise<FirefoxRDPAddonActor> {
     if (this.checkedForAddonReloading) {
       // We only need to check once if reload() is supported.
-      return Promise.resolve(addon);
+      return addon;
```

Comment:
I don't think `checkForAddonReloading` needs to return `addon` anymore. IIRC this was just to make chaining promises more convenient.